src/axis.ts

```diff
@@ -303,7 +303,8 @@ const COMMON_AXIS_PROPERTIES_INDEX: Flag<keyof (VgAxis | Axis)> = {
 
 const AXIS_PROPERTIES_INDEX: Flag<keyof Axis> = {
   ...COMMON_AXIS_PROPERTIES_INDEX,
-  encoding: 1
+  encoding: 1,
+  formatType: 1
```

Comment:
Doesn't Vega now have `formatType`?

Reply:
Yes, it does. This should not be here. Good catch!